runtime/service/src/test/java/org/apache/polaris/service/quarkus/catalog/AbstractIcebergCatalogTest.java

```diff
@@ -301,6 +289,10 @@ public void before(TestInfo testInfo) {
             storageCredentialCache,
             createEntityCache(polarisContext.getRealmConfig(), metaStoreManager));
 
+    // LocalPolarisMetaStoreManagerFactory.bootstrapServiceAndCreatePolarisPrincipalForRealm sets
+    // the CallContext.setCurrentContext() but never clears it, whereas the NoSQL one resets it.
```

Comment:
There is no NoSQL one right now, so perhaps better not to mention it in a comment here